qa/tasks/cephfs/test_misc.py

```diff
@@ -216,6 +216,20 @@ def test_fs_lsflags(self):
         self.assertEqual(lsflags["allow_multimds_snaps"], True)
         self.assertEqual(lsflags["allow_standby_replay"], True)
 
+    def test_dispatch_queue_throttle_message(self):
+        """
+        That cluster log a warning when the Dispatch Queue Throttle Limit hits
+        """
+        self.config_set('global', 'ms_dispatch_throttle_bytes', 500)
+        self.config_set('global', 'ms_dispatch_throttle_log_interval', 5)
+        self.config_set('global', 'ms_dispatch_throttle_clog_interval', 5)
+        # Create files & split across 10 directories, 1000 each.
+        with self.assert_cluster_log("Throttler Limit has been hit. "
+                                     "Some message processing may be significantly delayed.",
+                                     invert_match=False, watch_channel="cluster"):
```

Comment:
In my test on vstart, I got this:

````
2022-02-17T17:07:16.158391+0000 mds.c [WRN] Throttler Limit has been hit. Some message processing may be significantly delayed. Additional info: 296/500 bytes used, 2087 messages throttled.                                                                                                                                                                                                  
2022-02-17T17:07:17.690095+0000 mds.c [WRN] Throttler Limit has been hit. Some message processing may be significantly delayed. Additional info: 460/500 bytes used, 3472 messages throttled.                                                                                                                                                                                                  
````

Two cluster messages from `mds.c` 1 second apart?